modules/extensions-spring-support/src/test/resources/validation/source-with-expression-config-ref.xml

```diff
@@ -12,18 +12,17 @@
                        dateOfBirth="1959-09-07T00:00:00"
                        dateOfDeath="2011-09-07T00:00:00-05:00"
                        money="0"
-                       recipe="recipes"
                        initialHealth="CANCER"
-                       finalHealth="CANCER"
-                       monthlyIncomes="monthlyIncomes"
-                       candidateDoors="candidateDoors"
-                       enemies="#[mel:[]]">
+                       finalHealth="CANCER">
+        <heisenberg:enemies/>
```

Comment:
by doing this you're no longer testing what this tests verifies. There's already another test for the inlining case

Reply:
the test is called `source-with-expression-config-ref.xml`, what was being tested before has nothing to do.